tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaApisTest.java

```diff
@@ -314,6 +319,56 @@ public void testConsumerListOffsetLatest() throws Exception {
         assertEquals(listOffsetResponse.responseData().get(tp).timestamp, Long.valueOf(0));
     }
 
+    /**
+     * Test the sending speed of fetch request when the readable data is less than fetch.minBytes.
+     */
+    @Test(timeOut = 60000)
+    public void testFetchMinBytes() throws Exception {
+        String topicName = "testMinBytesTopic";
+        TopicPartition tp = new TopicPartition(topicName, 0);
+
+        // create partitioned topic.
+        admin.topics().createPartitionedTopic(topicName, 1);
+        List<TopicPartition> topicPartitions = new ArrayList<>();
+        topicPartitions.add(tp);
+
+        int maxResponseBytes = 800;
+        int maxPartitionBytes = 900;
+        int maxWaitMs = 3000;
+        //case1: consuming an empty topic
```

Comment:
Add space for comment.

```suggestion
        // case1: consuming an empty topic
```